### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by dolgopolovwork**
Just new readme and little refactor.

### CODE REVIEW

**Review by Alexey-N-Chernyshov — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#Chain adapter
+## Service overview 
+Chain adapter is a service that is used to fetch Iroha blocks very safely. Unlike the default Iroha blockchain listener implementation, the service is capable of publishing missing blocks.
```
Comment (LiraLemur) [inline README.md]: I'd say "service used for fetching"

File: README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#Chain adapter
+## Service overview 
+Chain adapter is a service that is used to fetch Iroha blocks very safely. Unlike the default Iroha blockchain listener implementation, the service is capable of publishing missing blocks. 
+
+The service is backed by RabbitMQ. All outgoing Iroha blocks are stored in the RabbitMQ exchange called `iroha` that fanouts all items to all bound queues. That implies that different services must use different queue names to avoid block stealing.
+
+Chain-adapter functionality may be used by utilizing the `ReliableIrohaChainListener` class. The class and the service itself are written using Kotling programming language entirely, but it has great interoperability with Java.
```
Comment (LiraLemur) [inline README.md]: Kotling?) 
I'd say: "The class and the service itself are written in Kotlin programming language that has great interoperability with Java."

Comment (dolgopolovwork) [reply README.md]: Oh, my bad. Thank you.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+The service is fail-fast, i.e it dies whenever Iroha or RabbitMQ goes offline.
+## Configuration file overview
+Chain adapter uses `chain-adapter.properties` as a default configuration file that is located in `resources` folder inside the project, however, every configuration item could be changed using environment variables.
```
Comment (LiraLemur) [inline README.md]: I'd make a separate sentence of this: "However, every configuration item could be changed through environmental variables.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+Chain-adapter functionality may be used by utilizing the `ReliableIrohaChainListener` class. The class and the service itself are written using Kotling programming language entirely, but it has great interoperability with Java.
+
+The service is fail-fast, i.e it dies whenever Iroha or RabbitMQ goes offline.
+## Configuration file overview
+Chain adapter uses `chain-adapter.properties` as a default configuration file that is located in `resources` folder inside the project, however, every configuration item could be changed using environment variables. 
+
+- `chain-adapter.rmqHost` - RabbitMQ host
+- `chain-adapter.rmqPort` - RabbitMQ port
+- `chain-adapter.irohaExchange` - exchange name that is used to publish blocks
+- `chain-adapter.lastReadBlockFilePath` - the file that chain adapter uses to save last read block height. This file is needed to publish missing blocks after a restart. It's important to highlight, that blocks that have lower height values won't be published.
```
Comment (LiraLemur) [inline README.md]: "the last read block height", "after restart",  "highlight that"

**Review by yurivin — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+- `chain-adapter.healthCheckPort` - health check port
+- `chain-adapter.queuesToCreate` - queues that chain adapter creates on a service startup. Queue names must be separated by the comma symbol. The value is optional.
+- `chain-adapter.iroha` - Iroha host and port configuration
+- `chain-adapter.dropLastReadBlock` - as it was mentioned before, chain adapter saves last read block height. It's possible to drop height to zero on a service startup by setting this value to `true`. Good for testing purposes.
```
Comment (LiraLemur) [inline README.md:20]: "the last read"

File: README.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+- `chain-adapter.rmqHost` - RabbitMQ host
+- `chain-adapter.rmqPort` - RabbitMQ port
+- `chain-adapter.irohaExchange` - exchange name that is used to publish blocks
+- `chain-adapter.lastReadBlockFilePath` - the file that chain adapter uses to save last read block height. This file is needed to publish missing blocks after a restart. It's important to highlight, that blocks that have lower height values won't be published.
+- `chain-adapter.healthCheckPort` - health check port
+- `chain-adapter.queuesToCreate` - queues that chain adapter creates on a service startup. Queue names must be separated by the comma symbol. The value is optional.
+- `chain-adapter.iroha` - Iroha host and port configuration
+- `chain-adapter.dropLastReadBlock` - as it was mentioned before, chain adapter saves last read block height. It's possible to drop height to zero on a service startup by setting this value to `true`. Good for testing purposes. 
+- `chain-adapter.irohaCredential` - credentials of an account that will be used by the service to listen to Iroha blocks. The account must have `can_get_blocks` permission.
```
Comment (LiraLemur) [inline README.md:21]: "of the account"

**Pull request merged**